reconcile/ocm_upgrade_scheduler_org_updater.py

```diff
@@ -48,10 +69,16 @@ def run(dry_run, gitlab_project_id):
                         logging.info(
                             ["add_cluster", ocm_name, ocm_cluster_name, default_name]
                         )
+                        policy = default["upgradePolicy"]
+                        if not policy:
+                            template = default["upgradePolicyTemplate"]
+                            policy = render_policy(
+                                template, ocm_cluster_spec, ocm_cluster_labels, settings
+                            )
```

Comment:
`upgradePolicyTemplate` is ignored when an `upgradePolicy` is defined. that might not be something obvious. could you add a `oneOf` declaration into the schema so bundling would fail if you provide both or none of them?

Reply:
i take it back. you have that in there already. i just missed it during my first review